blog/2024-09-16-ratify-1.3.0.mdx

```diff
@@ -0,0 +1,66 @@
+---
+slug: ratify-v1.3.0-is-available
+title: Ratify v1.3.0 is available! 
+authors: [ratify]
+tags: [verification]
+---
+
+We are excited to announce the release of [Ratify v1.3.0](https://github.com/ratify-project/ratify/releases/tag/v1.3.0)!
+
+Ratify is now a CNCF sandbox project! We sincerely thank the exceptional community for invaluable contributions and unwavering support in reaching this significant milestone.
+Ratify is a verification framework that ensures container images and supply chain artifacts stored in a registry are thoroughly vetted and trustworthy. 
+By [integrating Ratify into your Kubernetes](https://notaryproject.dev/blog/2023/announcing-major-release/#integration-with-admission-controller-for-kubernetes-usage) workflow, 
+you can safeguard your application deployment against potential threats and vulnerabilities, thereby greatly enhancing the security of your applications.
+
+Read further to learn about the exciting new features and how they can be beneficial to you!
+
+## Features and deprecations
+
+Key features in this release include:
+- Support of validating Notary Project signatures with timestamping
+- Support of periodic retrieval of keys and certificates stored in a KMS (Key Management System)
+- Improving error messages for artifact verification

```

Comment:
I think error message improvement doesn't belong to new features. It would be much consistent to align with the release notes https://github.com/ratify-project/ratify/releases/tag/v1.3.0

Reply:
Error message improvement can be a key enhancement, so it was listed here, but the current title used "feature" not "enhancement", I will follow the current convention to list error message improvements to "other enhancements"